corehq/apps/app_manager/templates/app_manager/v2/partials/case_config_shared.html

```diff
@@ -169,9 +169,21 @@ <h4 class="panel-title panel-title-nolink">
                             afterRender: $root.makePopover
                         }"></div>
                         <p class="help-block" data-bind="html: validate, visible: validate"></p>
-                        <div class="property-description"
-                          data-bind="text: $data.description, attr: {'data-content': $data.description}">
-                        </div>
+                        <!-- ko if: $data.description -->
+                        <inline-edit params="
+                            value: $data && $data.description,
```

Comment:
using `foo && bar` to return `bar` seems like a weird construct, but maybe just because i haven't encountered it before and initially thought it would return a boolean. but isn't the `$data &&` unnecessary anyway because this is inside a `if: $data.description`?